apps/researcher/src/lib/community/actions.ts

```diff
@@ -101,7 +104,26 @@ export async function getMyCommunities({
 
   const organizations = memberships.map(membership => membership.organization);
 
-  const communities = organizations.map(organizationToCommunity);
+  const communities = await Promise.all(
+    organizations.map(async organization => {
+      if (includeMembersCount) {
+        try {
+          const members =
+            await clerkClient.organizations.getOrganizationMembershipList({
+              organizationId: organization.id,
+            });
+          return organizationToCommunity({
+            ...organization,
+            members_count: members.length,
+          });
+        } catch (error) {
```

Comment:
Minor: use `(err)`, to be consistent with other catches, e.g. [1] and [2]?

[1] https://github.com/colonial-heritage/colonial-collections/blob/22ce5439e66b821c6907ab0dc743ad6229481c49/apps/researcher/src/components/add-object-list-form/index.tsx#L56
[2] https://github.com/colonial-heritage/colonial-collections/blob/22ce5439e66b821c6907ab0dc743ad6229481c49/apps/researcher/src/app/%5Blocale%5D/communities/page.tsx#L49